doc/indexing.rst

```diff
@@ -229,7 +229,7 @@ arrays). However, you can do normal indexing with dimension names:
 Using indexing to *assign* values to a subset of dataset (e.g.,
 ``ds[dict(space=0)] = 1``) is not yet supported.
 
-Dropping labels
+Dropping labels and dimensions
 ---------------
```

Comment:
I think you need to increase the width of the `----` line to match the new label.

Reply:
Done.